src/ui/simulator/windows/inspector/data.h

```diff
@@ -70,7 +70,7 @@ class InspectorData final
 
 public:
     //! Reference to the study
-    Data::Study& study;
+    Data::Study::Ptr study;
```

Comment:
Same as above...

Reply:
Same answer as above : it's safer not to de-reference smart pointers.

Reply:
To be precise, it is safeto de-reference for immediate use. Storing address is unsafe.